Dockerfile

```diff
@@ -0,0 +1,30 @@
+# Set the base image to Ubuntu
+FROM mysql
+
+# File Author / Maintainer
+MAINTAINER  Vivek/Denny
+
+# Update the repository sources list
+RUN apt-get update && apt-get install vim -y && apt-get install wget -y &&  mkdir /root/bot
+
+RUN cd /root/bot;wget https://raw.githubusercontent.com/vivekgrover1/jenkinsbot/master/python_mysql.py && \
+ wget https://raw.githubusercontent.com/vivekgrover1/jenkinsbot/master/slack_cmd_process.py && \
+ wget https://raw.githubusercontent.com/vivekgrover1/jenkinsbot/master/slack_message.py && \
+ wget https://raw.githubusercontent.com/vivekgrover1/jenkinsbot/master/slackbot.py && \
+ wget https://raw.githubusercontent.com/vivekgrover1/jenkinsbot/master/start_app.py && \
+ wget https://raw.githubusercontent.com/vivekgrover1/jenkinsbot/master/start_bot.sh
+
+
+ENV SLACK_BOT_TOKEN="your_bot_token" CHATBOT_NAME="your_bot_name" \
```

Comment:
We might also provide docker-compose.yml.

Thus people can use docker-compose to start this service in an organized way.